private/pdk-project.ts

```diff
@@ -145,6 +149,33 @@ export class PDKProject extends JsiiProject {
 
     this.pdkRelease = new PDKRelease(this);
   }
+
+  /**
+   * Provides the ability to override project specific NX Project Targets.
+   *
+   * @return Nx ProjectTargets specific to this package.
+   */
+  public getNxProjectTargets(): ProjectTargets {
+    const relativeDir = `${this.outdir.split(this.root.outdir)[1]}`;
```

Comment:
Is there reason not to use nx built-in var`{projectRoot}`? - https://nx.dev/more-concepts/customizing-inputs

Reply:
DIdn't know this existed! Will refactor :)